### PR TITLE
Adds support for EDDSA signatures

### PR DESCRIPTION
**Pull request by mz2**
Adds support for EDDSA signing Sparkle builds.

### CODE REVIEW

**Comment by jp-andre** [comment]
Thanks for your contribution @mz2. Please make sure that the spec tests pass.

File: lib/fastlane/plugin/appcenter/helper/appcenter_helper.rb [hunk 1]
```diff
@@ -409,12 +409,13 @@ def self.update_release(api_token, owner_name, app_name, release_id, release_not
       end
 
       # updates release metadata
-      def self.update_release_metadata(api_token, owner_name, app_name, release_id, dsa_signature)
+      def self.update_release_metadata(api_token, owner_name, app_name, release_id, dsa_signature, eddsa_signature)
         return if dsa_signature.to_s == ''
```
Comment (ajaykn) [inline lib/fastlane/plugin/appcenter/helper/appcenter_helper.rb:413]: I think, this condition needs to be updated with eddsa_signature as well.

**Comment by jp-andre** [comment]
@mz2 if you are still interested in getting this PR merged, please consider the above comments.

**Comment by kkyryl** [comment]
@jp-andre, @ajaykn I have submitted a similar pull request which accounts for your comments above. Please review if possible since it's blocking us from using App Center (we are migrating from a self hosted solution and we need EdDSA to work)

**Comment by DmitriyKirakosyan** [comment]
Closing as stale.

**Pull request closed without merging**